ios/FluentUI/DrawerVnext/SlideOverPanel.swift

```diff
@@ -76,6 +76,40 @@ struct SlideOverPanel<Content: View>: View {
         }
     }
 
+    private var resolvedBackgroundOpacity: Double {
```

Comment:
You can return the Colors directly in those computed properties:

- backgroundColor
- shadowColor

Reply:
I have added two separate properties instead of parametrizing them because they have a slightly different behavior.